File: 2020_ITE1014/6-2/4.py
a, b, c = input().split()
a = int(a)
b = int(b)
c = int(c)
if a >= b:
    if a >= c:
        if b >= c:
            print('min: ' + str(c))
            print('max: ' + str(a)) # a >= b >= c
        else:
            print('min: ' + str(b))
            print('max: ' + str(a)) # a >= c > b
    else:
        print('min: ' + str(b))
        print('max: ' + str(c)) # c > a >= b

else: # b > a
    if b >= c:
        if c >= a:
            print('min: ' + str(a))
            print('max: ' + str(b)) # b >= c >= a
        else:
            print('min: ' + str(c))
            print('max: ' + str(b)) # b > a > c
    else:
        print('min: ' + str(a))
        print('max: ' + str(c)) # c > b > a

File: 2020_ITE1014/6-2/4.c
#include <stdio.h>
int main()
{
	int a, b, c;
	scanf("%d %d %d", &a, &b, &c);
	if(a >= b)
	{
		if(a >= c)
		{
			if(b >= c)
			{
				printf("min: %d\n", c);
				printf("max: %d\n", a); //a >= b >= c
			}
			else
			{
				printf("min: %d\n", b); 
                                printf("max: %d\n", a); //a >= c > b
			}
		}
		else
		{
                                printf("min: %d\n", b); 
                                printf("max: %d\n", c); //c > a >= b
		}
	}
	else
	{
                if(b >= c)
                {

                        if(c >= a)
                        {
                                printf("min: %d\n", a); 
                                printf("max: %d\n", b); //b >= c >= a
                        }
                        else
                        { 
                                printf("min: %d\n", c); 
                                printf("max: %d\n", b); //b > a > c
                        }
		}
		else
		{
			printf("min: %d\n", a);
			printf("max: %d\n", c); //c > b > a
		}
	}
	return 0;
}
	
